Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -825,8 +826,10 @@ public static void HandleWeapons(Session session, params string[] parameters)
             {
                 foreach (uint weenieId in weaponsTest)
                 {
-                    WorldObject loot = LootGenerationFactory.CreateTestWorldObject(session.Player, weenieId);
+                    AceObject lootAceObject = (AceObject)DatabaseManager.World.GetAceObjectByWeenie(weenieId).Clone(GuidManager.NewItemGuid().Full);
+                    WorldObject loot = new GenericObject(lootAceObject);
```

Comment:
I tried that first - but it dies not look like it really works.   If you try to persist it, you will see that what you are doing behind the scenes, is making a world object which is made from an aceObject.   Then you are giving in a new guid.   The problem is that all of the aceObject contained in your new world object have the aceobjectid of the item you cloned from.   It does not matter as long as you are just spawning something that will never be persisted.   If you actually try to save that to the database, it blows up.   This actually works as coded.